src/main/kotlin/com/example/eventproject/service/WeatherService.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.eventproject.service
+
+import java.time.LocalDate
+
+interface WeatherService {
+
+    fun getCurrentTemperature(city: String): Double
```

Comment:
Que tal especificar que a temperatura retornada está em Kelvin e que a busca é por cidade, tipo `getCurrentTemperatureInKelvinByCity`?

Para refletir: como deixar o mesmo nome de função e ainda assim deixar explícito o tipo do parâmetro e do retorno para evitar erros de quem for usar?

Reply:
Estou nesse capítulo do Clean Code hahaha